### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by ValenciaWine**

### CODE REVIEW

**Comment by labiagajaime** [comment]
Rename your files like this "usoki.extension"

**Comment by ValenciaWine** [comment]
Ok thanks

File: Assignment2/Salvador Gonzalez Garcia/graph.rdf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	<rdf:Description rdf:about="#Sensor029">
+		<sensor:hasMeasurement rdf:resource="Measurement8401"/>
+	</rdf:Description>
+
+	<rdf:Description rdf:about="#Computer101">
+		<computer:hasOwner rdf:resource="User10A"/>
+	</rdf:Description>
+
+	<rdf:Description rdf:about="#Measurement8401">
+		<measurement:hasTemperature rdf:Integer="24"/>
```
Comment (labiagajaime) [inline Assignment2/Salvador Gonzalez Garcia/graph.rdf]: The correct value is 29

Comment (ValenciaWine) [reply Assignment2/Salvador Gonzalez Garcia/graph.rdf]: Indeed.

**Comment by idafensp** [comment]
Great conversation indeed! Interaction between students is welcome and taken into consideration.

File: Assignment2/Salvador Gonzalez Garcia/graph.rdf (hunk 1 above)
Comment (idafensp) [reply Assignment2/Salvador Gonzalez Garcia/graph.rdf]: is rdf:Integer a valid property? You should better use a XSD type to your value.

Comment (labiagajaime) [reply Assignment2/Salvador Gonzalez Garcia/graph.rdf]: same applies to me

Comment (idafensp) [reply Assignment2/Salvador Gonzalez Garcia/graph.rdf]: yes

Comment (ValenciaWine) [reply Assignment2/Salvador Gonzalez Garcia/graph.rdf]: `“29”^^xsd:integer`
I think this could be it.

**Pull request merged**